python/ray/rllib/__init__.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from ray.tune.registry import register_trainable
+from ray.rllib import ppo, es, dqn, a3c
+from ray.rllib.agent import _MockAgent, _SigmoidFakeData
+
+register_trainable("PPO", ppo.PPOAgent)
```

Comment:
In general, do these `register_trainable` calls need to happen on all workers? Or just on the driver?

Reply:
Just on the driver.